### ISSUE TITLE
Infinite loop caused by `Layout/ArgumentAlignment`, `Layout/FirstArgumentIndentation`, `Layout/LineLength` cops

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#14856]
## Steps to reproduce the problem

Run RuboCop with autocorrect on the following code:

```ruby
def validate_by_password
  case @login_error
  when :remaining_attempts_2, :remaining_attempts_1, :final_attempt
    attempts = Canvas::Security::LoginRegistry::WARNING_ATTEMPTS[@login_error]
    if @login_error == :final_attempt
      errors.add(password_field,
                 I18n.t("We've received several incorrect username or password entries. To protect your account, it has been locked. Please contact your system administrator."))
    else
      errors.add(password_field,
               I18n.t(
                   'Please verify your username or password and try again. After %<attempts>s more attempt(s), your account will be locked.', attempts:
                 ))
    end
  else
    errors.add(password_field,
               I18n.t('Login has been denied for security reasons. Please try again later or contact your system administrator.'))
  end
  nil
end
```

## Expected behavior

It should not enter an infinite correction loop.

## Actual behavior

> Infinite loop detected in file.rb and caused by Layout/ArgumentAlignment, Layout/FirstArgumentIndentation, Layout/LineLength

--------

## RuboCop version

```
$ bundle exec rubocop -V
1.84.1 (using Parser 3.3.10.0, rubocop-ast 1.49.0, analyzing as Ruby 2.7, running on ruby 3.4.8) [x86_64-linux]
  - rubocop-performance 1.26.1
  - rubocop-rake 0.7.1
  - rubocop-rspec 3.9.0
```

**Comment by koic** [comment]
This issue is a dup with #14858.
